dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/utils/UrlUtils.java

```diff
@@ -23,6 +23,7 @@
 public class UrlUtils {
     public static int getIdleTimeout(URL url) {
         int heartBeat = getHeartbeat(url);
+		// why the default value is heartBeat*3 ? because of the retry if client.
```

Comment:
Pls format your doc.
It seems that your indentation is not standardized.
In addition, I suggest you explain the meaning of the code directly rather than asking yourself.